bin/p2-preparer/main.go

```diff
@@ -44,6 +44,12 @@ func main() {
 		defer statusServer.Close()
 	}
 
+	if preparerConfig.RequireFile != "" {
```

Comment:
I don't think we want these six lines at all, though I could be wrong.

Because the p2-preparer is (in our deployment) started by p2-exec (which already checks the required file), it doesn't seem useful for p2-preparer to do a separate check. Or are we explicitly trying to support other environments in which p2-preparer isn't started by p2-exec?